OpenRA.Mods.Common/Widgets/Logic/Ingame/GameInfoStatsLogic.cs

```diff
@@ -85,7 +85,7 @@ public GameInfoStatsLogic(Widget widget, World world, OrderManager orderManager)
 				}
 
 				var team = item.Get<LabelWidget>("TEAM");
-				var teamNumber = pp.PlayerReference.Playable ? ((client == null) ? 0 : client.Team) : pp.PlayerReference.Team;
+				var teamNumber = pp.PlayerReference.Playable ? client.Team : pp.PlayerReference.Team;
```

Comment:
My assumption was that `client` can't be null once we're in a game, but #6444 seems to prove me wrong. I'll add null check before calling `SetupClientWidget` instead.